saleor/graphql/account/types.py

```diff
@@ -17,6 +17,8 @@
 from ..core.utils import get_node_optimized
 from ..decorators import one_of_permissions_required
 from ..utils import format_permissions_for_display
+from ..wishlist.resolvers import resolve_wishlist_items_from_user
```

Comment:
I am not confident that mixin resolvers between modules is the best idea. Maybe it should be places in `account.resolvers`?

Reply:
The resolver contains logic specific to the wishlist. I think it's ok to keep it separate, even though it's exposed under the `me {...}` endpoint in the `account` module.